tests/php/core-api/wpcom-fields/test-post-fields-publicize-connections.php

```diff
@@ -0,0 +1,294 @@
+<?php
```

Comment:
This file seems to use spaces instead of tabs for indentation - could we fix that?

Reply:
I took care of this.